testdata/import/BUILD

```diff
@@ -0,0 +1,45 @@
+load("//container:container.bzl", "container_import")
```

Comment:
Could you explain your reasoning why you think this is worth splitting off into a separate file?

Reply:
(also, if this is to be split off, it needs a license declaration)

Reply:
If I didnt move anything in testdata, this is the error I get:

```
ERROR: file 'testdata/da7bd81140ca921a067c604a3ba3f54d4e0d51ba5276cd9f32ad6a28e588f470.tar.gz.sha256' is generated by these conflicting actions:
Label: //testdata:pause_piecemeal_gz, //testdata:pause_piecemeal
RuleClass: container_import rule
Configuration: 7d61bd6adacde84bec39dad107f12252
Mnemonic: SHA256
Action key: cc9de884bd3f2c8c6e87d3ef4136c7ed, 205c201b5fd9a47a9f8ac944aff3562b
Progress message: SHA256 testdata/da7bd81140ca921a067c604a3ba3f54d4e0d51ba5276cd9f32ad6a28e588f470.tar.gz.sha256
PrimaryInput: File:[[<execution_root>]bazel-out/host/bin]external/bazel_tools/tools/build_defs/hash/sha256
PrimaryOutput: File:[[<execution_root>]bazel-out/k8-fastbuild/bin]testdata/da7bd81140ca921a067c604a3ba3f54d4e0d51ba5276cd9f32ad6a28e588f470.tar.gz.sha256
Owner information: //testdata:pause_piecemeal_gz BuildConfigurationValue.Key[7d61bd6adacde84bec39dad107f12252] false, //testdata:pause_piecemeal BuildConfigurationValue.Key[7d61bd6adacde84bec39dad107f12252] false
MandatoryInputs: Attempted action contains artifacts not in previous action (first 5): 
	testdata/da7bd81140ca921a067c604a3ba3f54d4e0d51ba5276cd9f32ad6a28e588f470.tar.gz
Previous action contains artifacts not in attempted action (first 5): 
	testdata/da7bd81140ca921a067c604a3ba3f54d4e0d51ba5276cd9f32ad6a28e588f470.tar.gz
Outputs: are equal
ERROR: Analysis of target '//testdata:pause_piecemeal_gz' failed; build aborted
```

Reply:
sgtm, see below. Also, please add license declaration to both new BUILD files

Reply:
Thanks for explaining, and sorry I missed the doc comment in the file below. sgtm to have them in separate folders. 
Could you please:
- create one folder for each of these files under testdata and give them a name that is more specific to what they declare (e.g., this one could be testdata/pause_piecemeal/BUILD the one below could be testdata/pause_piecemeal_gz/BUILD, or something like that)
- add license declarations doc comments to both files (see other BUILD files for reference)
- add doc comment in both BUILD files explaining why they need to be separated (right now doc is only in the file below)